errbot/backends/irc.py

```diff
@@ -254,7 +267,7 @@ def occupants(self):
         occupants = []
         try:
             for nick in self._bot.conn.channels[self.room].users():
-                occupants.append(IRCMUCOccupant(nick=nick, room=self.room))
+                occupants.append(IRCMUCOccupant(nick, self.room))
```

Comment:
Keeping keyword arguments here would actually be safer (and more explicit, thus more clear)